tests/tests/lightspeed-install.cy.ts

```diff
@@ -18,21 +18,24 @@ const minimizeButton = `${popover} .ols-plugin__popover-control[title=Minimize]`
 const expandButton = `${popover} .ols-plugin__popover-control[title=Expand]`;
 const collapseButton = `${popover} .ols-plugin__popover-control[title=Collapse]`;
 const clearChatButton = `${popover} .ols-plugin__popover-clear-chat`;
-const userChatEntry = `${popover} .ols-plugin__chat-entry--user`;
-const aiChatEntry = `${popover} .ols-plugin__chat-entry--ai`;
-const attachments = `${popover} .ols-plugin__chat-prompt-attachments`;
-const attachMenuButton = `${popover} .ols-plugin__attach-menu`;
-const attachMenu = `${popover} .ols-plugin__context-menu`;
+const userChatEntry = `${popover} .pf-chatbot__message--user`;
```

Comment:
Nothing to change in this PR. But we prefer to use test-ids instead as style classes could change. We are aligning all the observability plugins with this approach.